src/internal/collection/postgres_schema.go

```diff
@@ -129,6 +193,13 @@ func SetupPostgresCollections(ctx context.Context, sqlTx *sqlx.Tx, collections .
 			return errors.EnsureStack(err)
 		}
 
+		for _, idx := range col.indexes {
```

Comment:
Unrelated: I noticed we weren't actually creating secondary indexes on the collections tables, so non-pkey queries were not efficient - fixed here.